Source/WebCore/Modules/applepay/ApplePaySessionPaymentRequest.h

```diff
@@ -195,16 +201,20 @@ class ApplePaySessionPaymentRequest {
 #endif
 
 #if ENABLE(APPLE_PAY_RECURRING_PAYMENTS)
-    std::optional<ApplePayAutomaticReloadPaymentRequest> m_automaticReloadPaymentRequest;
+    std::optional<ApplePayRecurringPaymentRequest> m_recurringPaymentRequest;
```

Comment:
Funny that these were the wrong way around. But it also makes me wonder if we need the ENABLE guards any more.

Reply:
Luckily those two features were added at the same time, so it just worked™. 😄